test/e2e/data/infrastructure-gcp/cluster-template-ci-gke.yaml

```diff
@@ -60,4 +60,7 @@ spec:
   scaling:
     minCount: ${GKE_MACHINE_POOL_MIN}
     maxCount: ${GKE_MACHINE_POOL_MAX}
-
+  kubernetesTaints:
+    - key: "key1"
+      operator: "Exists"
```

Comment:
Where does this `operator` field come from, I cannot find it in `Taint` definition: https://github.com/kubernetes-sigs/cluster-api-provider-gcp/blob/ab66a2c8acee99e1a13ea4afaefeb18e7b3ada62/exp/api/v1beta1/types.go#L29

Reply:
I see https://doc.crds.dev/github.com/kubernetes-sigs/cluster-api-provider-gcp/infrastructure.cluster.x-k8s.io/GCPManagedMachinePool/v1beta1@v1.8.0#spec-kubernetesTaints 🤦 I took the one from the definitions in k8s doc https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/#concepts

Reply:
```suggestion
      value: "value1"
```